test_scripts/API/RAI/004_ATF_N_correlationID_IsMissed_RAI_steeringWheelLocation.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [RegisterAppInterface] Conditions for SDL to provide the default value of "steeringWheelLocation" param to each app in response
+-- [HMI_capabilities] The 'hmi_capabilities' struct
+-- [MOBILE_API] [HMI_API] The 'steeringWheelLocation' enum
+-- [MOBILE_API] The 'steeringWheelLocation' param
+--
+-- Description:
+-- In case SDL does NOT receive value of "steeringWeelLocation" parameter via UI.GetCapabilities_response from HMI
+-- SDL must retrieve the value of "steeringWeelLocation" parameter from "HMI_capabilities.json" file and 
+-- provide the value of "steeringWeelLocation" via RegisterAppInterface_response to mobile app
+--
+-- 1. Used preconditions
+-- In InitHMI_OnReady HMI replies with correlationID missing in json message to UI.GetCapabilities
+-- SDL invalidates message
+-- Get value of "steeringWeelLocation" parameter from "HMI_capabilities.json".
+--
+-- 2. Performed steps
+-- Register new application.
+--
+-- Expected result:
+-- SDL->mobile: RegisterAppInterface_response steeringWeelLocation is provided equal to "HMI_capabilities.json"
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local testCasesForRAI = require('user_modules/shared_testcases/testCasesForRAI')
+local events = require("events")
+local mobile_session = require('mobile_session')
+
+--[[ Local variables ]]
+local value_steering_wheel_location = testCasesForRAI.get_data_steeringWheelLocation()
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForRAI:Connecttest_onlywith_StartSDL_InitHMIOnReady("connecttest_initHMI.lua")
```

Comment:
@OVikhrov, yes it is removed because of comments from aderiabin, Now it static user_modules/connecttest_initHMI.lua is used.
Done in bce0a269222aa710ec4a4d716f3eaabfd1cb5afe